compiler/src/iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -751,6 +755,51 @@ static bool isIdentityMapWithZeros(AffineMap map) {
   return dimsSeen == map.getNumDims();
 }
 
+/// For the fusion of root op -> elementwise operation to be bufferized
+/// in-place without use of extra memory, the result of the root operation
+/// must be able to reuse the buffer for the result of the elementwise
+/// operation. This is possible if input and output are accessed using the same
+/// indexing map.
+// TODO: This restriction can go away if we can vectorize always, but that has
+// a long tail of tasks.
+static bool isInsOperandBufferizable(OpOperand *insOperand,
+                                     bool aggressiveFusion) {
+  // Ignore the check if in-place bufferization is not required.
+  if (!clEnsureInplaceableConsumer) return true;
+
+  auto linalgOp = dyn_cast<linalg::LinalgOp>(insOperand->getOwner());
+  if (!linalgOp) return false;
+
+  AffineMap insOperandIndexingMap = linalgOp.getTiedIndexingMap(insOperand);
+
+  auto canTieWithOutsOperand = [&](OpOperand *outsOperand) {
+    AffineMap outsOperandIndexingMap = linalgOp.getTiedIndexingMap(outsOperand);
+
+    if (outsOperandIndexingMap != insOperandIndexingMap) {
+      // if (!aggressiveFusion) return false;
```

Comment:
Do we want to keep this check? It's to make sure that when aggressiveFusion is disabled, we won't generate stack buffer.

Reply:
we have a compile time check for large allocations. So whatever is within that limit

Reply:
Then I guess we can remove the argument `aggressiveFusion` from `isInsOperandBufferizable`?